vignettes/Life_Data_Analysis_Part_II.md

```diff
@@ -0,0 +1,320 @@
+than ML estimators.
+
+[^note3]: Genschel, U.; Meeker, W. Q.: _A Comparison of Maximum Likelihood and Median-Rank Regression for Weibull Estimation_, 
+          in: _Quality Engineering 22 (4)_, DOI: 10.1080/08982112.2010.503447, 2010, pp. 236-255
+
+
+## Data I: shock
+
+To apply the introduced estimation methods and related visualizations the `shock` data taken from `SPREDA` package is used. In this dataset kilometer-dependent problems that occurred on shock absorbers are reported. In addition to failed items the 
+dataset also contains non-defectives. The data can be found in _Statistical Methods for Reliability Data_ [^note4]. 
+
+[^note4]: Meeker, W. Q.; Escobar, L. A.: _Statistical Methods for Reliability Data_, 
+          _New York, Wiley series in probability and statistics_, 1998, p. 630  
+          
+## Median Rank Regression and Maximum Likelihood Estimation with Package `weibulltools`
+
+Where `rank_regression()` determines a regression model (`x ~ y`) based on the linearized CDF of a specified model, `ml_estimation()` computes the parameters which maximize the log-likelihood of an underlying distribution. Both methods can be applied to complete failure data as well as failure and (multiple) right-censored data. A special characteristic of these functions is that the implemented distributions are parameterized in such a way that they are part of a (log-)location-scale family with parameters $\mu$ and $\sigma$. This representation is often used in reliability analysis. Subsequent transformations especially to get the Weibull-specific values in the scale-shape parameterization with $\eta$ and $\beta$ are included. This parameterization is also used in the `stats` package (e.g., in the function [`pweibull`](https://stat.ethz.ch/R-manual/R-devel/library/stats/html/Weibull.html)). `rank_regression()` and `ml_estimation()` can also deal with models that have a threshold parameter $\gamma$ like the three-parametric Weibull. If a three-parametric distribution is specified both methods call corresponding profiling functions. Inside `rank_regression()` the function `r_squared_profiling()` is called and if `ml_estimation()` is used `loglik_profiling()` is executed. To get information about the provided confidence intervals of the parameters the help pages of `rank_regression()` and `ml_estimation()` should be read carefully.  
+
+In the following we want to apply both methods to the dataset `shock`. We assume that the data can be best described by a two-parametric Weibull distribution. The estimated parameters will be used to calculate the population lines and for the purpose of comparison they will be visualized in one Weibull probability plot. 
+
+### MRR-Code two-parametric Weibull
+
+
+```r
+library(SPREDA) # for dataset shock
```

Comment:
Not needed anymore

Reply:
All your comments show the need to rework the vignettes. But we should first finish the other stuff